### DISCUSSION TITLE
[request feature]Could add dark color theme/mode to strawberry?

### DISCUSSION THREAD

**Discussion by aknyt** [strawberrymusicplayer/strawberry#1449] [Ideas]
For modern desktop of linux distro, I use dark  theme/mode usually. Strawberry/Clementine does not have dark mode. Feel not unified in dark mode desktop theme.

Could add dark mode/theme feature to strawberry?

**Comment by guprobr** [comment]
search for this: **qt5ct** or **qt6ct** programs....
you can customize the look and feel of Qt stuff with it :D
colors, fonts, sizes, even make css for the widgets and break stuff like hell!
:dancing_women:
